mmv1/third_party/terraform/services/spanner/resource_spanner_database_test.go.erb

```diff
@@ -604,6 +604,141 @@ resource "google_project_service_identity" "ck_sa" {
   service  = "spanner.googleapis.com"
 }
 
+`, context)
+}
+
+func TestAccSpannerDatabase_mrcmek(t *testing.T) {
+	acctest.SkipIfVcr(t)
```

Comment:
Why is this skipped in VCR? Use of KMS?

Did we verify it passing locally?

Reply:
Yeah I noticed that the other CMEK test above was also skipping VCR.